### ISSUE TITLE
What still needs to be done

### ISSUE THREAD

**Issue by MaxNeedsSnacks** [MaxNeedsSnacks/AdAstra#29]
[ ] Advanced Mechanistics - Chapter (EnderIO, Mekanism, early-game AE2 (no Controller), Extended Crafting pt. 1, Modular Machinery)
[ ] Advanced Mechanistics - Quests (most EnderIO and Mekanism quests, AE2 Energy Acceptor, Crafting Core + Pedestals)
[ ] Advanced Mechanistics - Resources (Mekanism Circuits, Platinum, Mithril, Crystalline Alloy, maybe Litherite?)

[ ] The Atomic Age - Chapter (AE2, end-game TechReborn, end-game Calculator)
[ ] The Atomic Age - Quests (some AE2 quests, Atomic and Flawless Calculator, most Advanced Tech Reborn stuff)
[ ] The Atomic Age - Resources (Chrome, Titanium, Iridium)

[ ] Botania / Astral / Blood Magic - Questline?
[ ] Botania / Astral / Blood Magic- Nerfs / Gating (I haven't even **touched** that yet)
[ ] Botania / Astral / Blood Magic- Resources (Some things like Mithril or Adamantium could require magic mods...)

What else still needs to be sorted:
[ ] Advanced Resources (stuff like Light and Dark Matter from Sky Resources or Tungstensteel)
[ ] Draconic Evolution (you can't even get to the end because you need Light and Dark Matter for that)
[ ] Environmental Tech
[ ] Utility Mods Questline (ExU, maybe add Cyclic back?, Dark Utilities)
[ ] End-game Extended Crafting, all of that end-game stuff in general

**Comment by MaxNeedsSnacks** [comment]
Yeah I think "help wanted" is appropriate here when I haven't even gotten any testers for the early-game of the pack yet 😅

**Comment by NillerMedDild** [comment]
Ouch, that's quite the to-do list :P

**Comment by MaxNeedsSnacks** [comment]
Oh yeah, definitely is.

still not quite as much as you need to do for Scientia tho :P
